src/subcommands/deploy.ts

```diff
@@ -0,0 +1,306 @@
+// Copyright 2021 Deno Land Inc. All rights reserved. MIT license.
+
+import { fromFileUrl, join, normalize, Spinner, wait } from "../../deps.ts";
+import { error } from "../error.ts";
+import { API, APIError } from "../utils/api.ts";
+import { ManifestEntry } from "../utils/api_types.ts";
+import { parseEntrypoint } from "../utils/entrypoint.ts";
+
+const help = `deployctl deploy
+Deploy a script with static files to Deno Deploy.
+
+To deploy a local script:
+  deployctl deploy --project=helloworld main.ts
```

Comment:
Since the project flag is required, maybe it should just be 
```
deployctl deploy helloworld main.ts
```

Reply:
I am hoping the project name will be in some config file in the future, at which point the flag wouldn't be needed anymore.

Reply:
Ok fair enough